documentation/smg/11_ISIS_release.md

```diff
@@ -1,79 +1,161 @@
-
-`/home/isis_direct_soft/Horace_git`
-`/home/isis_direct_soft/Horace_bugfix`
-`/home/isis_direct_soft/Herbert_git`
-`/home/isis_direct_soft/Herbert_bugfix`
+#### Contact
 
-By agreement, this location is accessed by code users through symbolic links, created in `/usr/local/mprogs` folder.
+IDAaaS support team : support@analysis.stfc.ac.uk
+
+#### Main Deploy Location
+
+Various versions and components of Horace are deployed in a number of local repositories under the path `/usr/local/mprogs`.
+
+The physical disk location of these Horace repositories is the same as for ISISCOMPUTE system; the description for that system below gives more details.
+As this may change subject to development of the underlying IDAaaS system structure, the path `usr/local/mprogs` provides symbolic links to the repositories' current locations.
+Note that the repository symbolic link names may not always match exactly the directory name of the underlying repository location.
+To ensure compatibility and smooth user's experience, all code operations should be performed with these symbolically linked paths, 
+which will remain constant regardless of the actuall physical location of the code.
+
+The principal repositories are
+- Horace (underlying name: Horace_git): this is the directory corresponding to `horace_core` in a clone of the full Horace github repository.
+- Herbert (underlying name: Herbert_git): this is the directory corresponding to `herbert_core` in a clone of the full Horace github repository.
+
+
+#### Main Deploy Process
+
+The version deployed is the one on the master branch of the github repository https://github.com/pace-neutrons/Horace, and the process of deployment for a simple fix is the same as for
+fixing or updating the master branch in any development process:
+- clone the repository, either to IDAaaS or your own local machine
+- create and checkout a new branch referencing a ticket for the changes
+- make the changes, and add them and commit them to your repository clone
+- push the changes, which should pass all tests and be approved by reviewers before merging into the master branch
+- if all tests pass and all reviewers agree, merge the pull request.
+
+The final additional step is to make the changes in github available on IDAaaS; so far they have only been made on github.
+- On IDAaaS, open a terminal and change directory to the Horace (Horace_git) repository named above.
+- Check the repository's status with the `git status` command, You should find that it is a git repository checked out on the `master` branch. 
+For other outputs, please consult with the Horace developers.
+- Bring the changes into the IDAaaS repository with the command `git pull`.
+- If you have also made changes in the `herbert_core` directory, repeat these steps in the Herbert (Herbert_git) repository. 
+Although Horace and Herbert have been merged into a single repository on github, they are stilll represented by two separate repositories on IDAaaS.
```

Comment:
old versions are still present on `IDAAAS until expected to be used with old versions of Horace

Reply:
@abuts As noted above on my comment on line 36, the mprogs/Horace slinks to repo Horace_git (its folder horace_core) which is the latest master or branch from it, while mprogs/Herbert slinks to a different repo Herbert_git (its folder herbert_core) which is branched to the v3.6.3 release. In general I am confused about how Horace and Herbert are paired up.